docs/application-connector/10-04-connector-service-troubleshooting.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Errors while generating or renewing a certificate
+type: Troubleshooting
+---
+
+## Invalid Certificate Signing Request (CSR)
+
+If you try to generate a client certificate, you may get this error:
+```json
+{
+  "code":403,
+  "error":"CSR: Invalid common name provided."
+}
+```
+
+This error is caused by applying wrong subject info to your Certificate Signing Request.  
+To ensure it was generated properly, check the values returned by the Connector Service with the call that fetched CSR info:
+```json
+{
+  "csrUrl":"https://connector-service.name.cluster.extend.cx.cloud.sap/v1/applications/certificates?token=5o7ucwjz9vcpFlBsHJcwnnuL-rU8af1MsfQ6OlWTgauw7aB-xtSkXUn_ts0RtMMKhvlZVPridqmAPbf2mKC8YA==",
```

Comment:
I don't think you need to include the entire response. What's the benefit of that?